tasks/agent.py

```diff
@@ -278,6 +278,20 @@ def run(
     config_path = os.path.join(BIN_PATH, "dist", "datadog.yaml") if not config_path else config_path
     ctx.run(f"{agent_bin} run -c {config_path}")
 
+@task
+def status(
```

Comment:
I wonder if we can/should make this generic (e.g. `inv agent.exec <subcommand> [<args>]`) to be a bit more flexible? Also, do we want to build/run the agent as a dep of this task?

PS: not a blocker but I wonder if these ancillary util helpers should go in a separate file

Reply:
its a good question, this problem is not specific to `status`, but rather any operation where you want to easily query the currently running agent using a subcommand. 

I think I like that actually, I'm going to switch this to `inv agent.exec <subcommand>`